frontend_tests/node_tests/pm_list.js

```diff
@@ -138,9 +224,13 @@ global.people.initialize_current_user(me.user_id);
 
     assert(collapsed);
     assert(bob_li.hasClass('active-sub-filter'));
+
+    narrow_state.active = function () { return false; };
+    pm_list.update_private_messages();
+
 }());
 
-(function test_update_dom_with_unread_counts() {
+  (function test_update_dom_with_unread_counts() {
```

Comment:
not intended. will remove. must have crept as result of collapsing code. I remember atom editior do some strange thing when I collapsed the code, now I know it added a space.